lib/lita/handlers/api/market.rb

```diff
@@ -25,10 +25,11 @@ def limit_orders(request, response)
         def place_limit_order(request, response)
           return respond_not_authorized(response) unless authorized?(request)
           user = current_user(request)
-          body = request.body.read
+          order = limit_order_for_user(user)
           if user
-            if winning_list.include?(user.mention_name) && market_manager.add_limit_order(body)
-              respond(response, success: true)
+            if winning_list.include?(user.mention_name) &&
+               market_manager.add_limit_order(order)
```

Comment:
Use 4 (not 3) spaces for indenting a condition in an if statement spanning multiple lines.